cfme/intelligence/chargeback/rates.py

```diff
@@ -402,5 +404,8 @@ class StorageRateEdit(CFMENavigateStep):
     VIEW = EditStorageChargebackView
     prerequisite = NavigateToSibling('Details')
 
+    def am_i_here(self):
```

Comment:
I think we should not use this directly... 
Q: Its not possible to improve `is_displayed` of all views. The good practice is to make `is_displayed` more unique for views.